### PR TITLE
[ASAN]Increasing switch create timeout for ASAN images

### PR DESCRIPTION
**Pull request by dgsudharsan**
**What I did**
When running ASAN images with low CPU systems, due to the additional overhead associated with ASAN, it results in switch create timeout as seen in logs below. Hence increasing the **switch create timeout to 2x for ASAN builds.** e.g. For regular switch scenario, the timeout will be 120 seconds  instead of 60 seconds.

```
Apr 18 20:23:46.749074 arc-switch1004 NOTICE swss#orchagent: :- create: request switch create with context 0
Apr 18 20:23:46.749074 arc-switch1004 NOTICE swss#orchagent: :- allocateNewSwitchObjectId: created SWITCH VID oid:0x21000000000000 for hwinfo: ''
Apr 18 20:24:46.816998 arc-switch1004 ERR swss#orchagent: :- wait: SELECT operation result: TIMEOUT on getresponse
Apr 18 20:24:46.817243 arc-switch1004 ERR swss#orchagent: :- wait: failed to get response for getresponse
Apr 18 20:24:46.817405 arc-switch1004 ERR swss#orchagent: :- create: create status: SAI_STATUS_FAILURE
Apr 18 20:24:46.817552 arc-switch1004 ERR swss#orchagent: :- main: Failed to create a switch, rv:-1

```

**Why I did it**
To avoid timeout to create switch when testing ASAN builds with lower CPU systems

**How I verified it**
Loaded the build with changes and verified no issues are seen

**Details if related**

### CODE REVIEW

**Review by oleksandrivantsiv — APPROVED** [review]

**Review by oleksandrivantsiv — APPROVED** [review]

**Review by oleksandrivantsiv — APPROVED** [review]

**Review by oleksandrivantsiv — APPROVED** [review]

File: orchagent/main.cpp [hunk 1]
```diff
@@ -585,7 +585,13 @@ int main(int argc, char **argv)
     attr.value.u64 = gSwitchId;
     attrs.push_back(attr);
 
+    auto delay_factor = 1;
+
+#ifdef ASAN_ENABLED
```
Comment (prsunny) [inline orchagent/main.cpp:590]: Prefer not use `#ifdefs `in code. @xumia do you've any suggestions?

**Review by prsunny — APPROVED** [review]
lgtm. @xumia to review as well.

**Review by oleksandrivantsiv — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@xumia  Can you please help to close the review?

**Review by xumia — APPROVED** [review]

**Comment by prsunny** [comment]
@dgsudharsan , can you please update the description as to what was the previous timeout and what is the new value in seconds?

**Comment by dgsudharsan** [comment]
> @dgsudharsan , can you please update the description as to what was the previous timeout and what is the new value in seconds?

Done

**Pull request merged**